src/containers/form/FormContainer.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import Promise from 'bluebird';
-import { EntityDataModelApi, DataApi, SyncApi } from 'lattice';
+import { AppApi, EntityDataModelApi, DataApi, SyncApi } from 'lattice';
```

Comment:
You'll have to upgrade the `lattice-js` dependency once we release a new version.